src/loaders/api/api.js

```diff
@@ -54,35 +54,17 @@ export function setAPI (agentIdentifier, forceDrain, runSoftNavOverSpa = false)
   var prefix = 'api-'
   var spaPrefix = prefix + 'ixn-'
 
-  /**
-   *
-   * @param {string} message
-   * @param {{[key: string]: *}} context
-   * @param {string} level
-   */
-  function log (message, customAttributes, level = 'info') {
-    handle(SUPPORTABILITY_METRIC_CHANNEL, [`API/logging/${level}/called`], undefined, FEATURE_NAMES.metrics, instanceEE)
-    handle('log', [now(), message, customAttributes, level], undefined, FEATURE_NAMES.logging, instanceEE)
-  }
-
   logApiMethods.forEach((method) => {
     apiInterface[method] = function (message, customAttributes = {}) {
-      log(message, customAttributes, method.toLowerCase().replace('log', ''))
+      bufferLog(instanceEE, message, customAttributes, method.toLowerCase().replace('log', ''))
     }
```

Comment:
Instead of exposing every log___ as api, why not just expose one `.log` and have the level be an arg? Idr if we talked about this and settled on one way or another.

Reply:
I understand, and kind of agree TBH -- BUT... this is a pattern i'd like to avoid that we have fallen victim to in the past:
-- plan
-- agree on plan
-- write docs, CDDs, briefs and get approval from product
-- THEN decide to change major decisions while writing the code

Lets talk more about this internally

Reply:
EM support gathered, will change to single API with arg